src/mainmenu.cpp

```diff
@@ -6,9 +6,10 @@ void MainMenu::logic(void)
 	//Determine size and position of Title Text
 	TitleFontSize = GetScreenHeight()/10;
 	TitleDims = MeasureTextEx(GetFontDefault(), "Geometry Slash", TitleFontSize, TitleFontSize/10);
-	TitlePos.x = GetScreenWidth()/2 - TitleDims.x/2;
+	TitlePos.x = GetScreenWidth()/2 - TitleDims.x/1.6;
```

Comment:
I think the proper change is to add the version number to the MeasureTextEx call